lib/config.py

```diff
@@ -88,7 +88,7 @@
     ('release-notes', 'Create release notes',
         [RELEASE_NOTES_ARGS, PUSH_REPO_ARGS, BUILD_REPO_ARGS]),
     ('upgrade-versions', 'Upgrade packages versions',
-        [PUSH_REPO_ARGS, BUILD_REPO_ARGS]),
```

Comment:
Please explain the motivation in the commit longer message. I believe you want to select the packages that are being upgraded?